.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
-/public
```

Comment:
Is it intentional? I believe `public` should be ignored?

Reply:
GH doesn't allow to comment on empty file, but could you remove empty .sh files from the PR?